src/v/redpanda/admin/api-doc/redpanda_services_restart.json

```diff
@@ -0,0 +1,33 @@
+"/v1/redpanda-services/restart": {
+  "put": {
+    "summary": "Restart a redpanda service.",
+    "operationId": "redpanda_services_restart",
+    "parameters": [
+      {
+        "name": "service",
+        "in": "query",
+        "required": true,
+        "type": "string"
+      }
+    ],
+    "produces": ["application/json"],
+    "responses": {
+      "200": {
+        "description": "Restart success",
+        "schema": "json"
+      },
+      "404": {
+        "description": "Service not found",
+        "schema":{
+            "type": "json"
+        }
+      },
+      "500": {
+        "description": "Internal Server error",
+        "schema": {
+          "type": "json"
+        }
```

Comment:
I'm not convinced these are valid swagger, and there's some inconsistency here (the first one is different to the others)

Reply:
Removed the schema portion as it wasn't necessary.